canonical-web-and-design/jaas-dashboard
# Move elements around and fix beta label spacing.

## Done

Move elements around and fix beta label spacing.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The beta label text should be centered in the button.

## Details

Fixes #469 

## Screenshots

![Screen Shot 2020-05-01 at 8 02 51 AM](https://user-images.githubusercontent.com/532033/80811112-7a2cff00-8b82-11ea-88bd-1537b9e65267.png)
